# HG changeset patch
# User Philip Nienhuis <prnienhuis@users.sf.net>
# Date 1479416095 -3600
#      Thu Nov 17 21:54:55 2016 +0100
# Node ID b25e41637412baa2e27da88f15f2f0c364df6058
# Parent  fadaacb3d816db0b90ab396605f938f2a4ed99c9
(pkg): restore accidentally dropped 'desc' arg (bug #49635)

diff --git a/scripts/pkg/private/default_prefix.m b/scripts/pkg/private/default_prefix.m
--- a/scripts/pkg/private/default_prefix.m
+++ b/scripts/pkg/private/default_prefix.m
@@ -16,17 +16,17 @@
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {} {[@var{prefix}, @var{archprefix} =} default_prefix (@var{global_install})
 ## Undocumented internal function.
 ## @end deftypefn
 
-function [prefix, archprefix] = default_prefix (global_install)
+function [prefix, archprefix] = default_prefix (global_install, desc)
 
   if (global_install)
     prefix = fullfile (OCTAVE_HOME (), "share", "octave", "packages");
     if (nargin == 2)
       archprefix = fullfile (__octave_config_info__ ("libdir"), "octave",
                              "packages", [desc.name "-" desc.version]);
     else
       archprefix = fullfile (__octave_config_info__ ("libdir"), "octave",
